### ISSUE TITLE
NTR: Prolactin response low after TRH stimulation (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8334]
New term request Prolactin response low after TRH stimulation (elthat_130419045432): Thyrotropin (thyrotrophin)-releasing hormone (TRH) resistance

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000241
- label: Diminished prolactin response to thyrotrophin-releasing hormone stimulation
- parents: [Abnormal TSH response to thyrotrophin-releasing hormone stimulation test (HP:0033080)](https://hpo.jax.org/app/browse/term/HP:0033080)
- pmid: PMID:6430949
- def: The increase in prolactin concentration (PRL) following stimulation with thyrotropin (thyrdoid releasing hormone) as compared to baseline PRL concentration is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8334
- synonyms:
